### ISSUE TITLE
python3 train_syncnet_sam.py

### ISSUE THREAD

**Issue by einsqing** [primepake/wav2lip_288x288#101]
Step 3355 | out_of_sync_distance: 0.50673163 | Loss: 0.56647605 | Elapsed: 0.77767
Step 3356 | out_of_sync_distance: 0.50098807 | Loss: 0.56791657 | Elapsed: 0.76504
Step 3357 | out_of_sync_distance: 0.50698453 | Loss: 0.56805406 | Elapsed: 0.72830
Step 3358 | out_of_sync_distance: 0.53348744 | Loss: 0.56974050 | Elapsed: 0.77868
Step 3359 | out_of_sync_distance: 0.48923862 | Loss: 0.56817545 | Elapsed: 0.75280

The expert discriminator's eval loss should go down to ~0.25 and the Wav2Lip eval sync loss should go down to ~0.2 to get good results.

Does the first step of loss also need to be trained to 0.25？Step 2 to 0.2？

**Comment by shahidmuneer** [comment]
I have reached a loss of ~ 0.29 it will go further down to 0.20, and then I will start training wav2lip_sam. So you should wait till the loss goes down to 0.2. 
Thanks

**Comment by fzuqxt** [comment]
@einsqing 你好请问下您这个口型判别器是怎么跑起来的，我卡在0epoch动不了

**Comment by fzuqxt** [comment]
我加载无反应

**Comment by zhuxiu1234** [comment]
我加载也无反应，有解决吗？

> 我加载无反应

**Comment by ashishjamarkattel** [comment]
may be its due to batch size being large enough or your dataloader is not sending data. Mostly its problem of dataloader.

**Comment by fqhwwlzzbh** [comment]
> 我加载也无反应，有解决吗？
> 
> > 我加载无反应

源代码里面有很多坑，你首先要看一下你的程序有没有读取到文件，没有读取到文件的话，会一直循环下去，作者给报错隐藏了，所以你也看不见报错。

**Comment by eminem171333491** [comment]
> 我加载无反应
我查出来是librosa版本高了
librosa==0.7.0，numba==0.48就行了

**Comment by jibingyangsf** [comment]
请问这套384 有在不改网络结构和损失函数情况下 收敛的很好的吗
